src/LiveComponent/assets/src/live_controller.ts

```diff
@@ -419,7 +419,7 @@ export default class extends Controller {
     /**
      * @private
      */
-    _handleLoadingDirective(element: HTMLElement, isLoading: boolean, directive: Directive) {
+    _handleLoadingDirective(element: HTMLElement|SVGElement, isLoading: boolean, directive: Directive) {
```

Comment:
Excellent!  I think `_getValueFromElement()` might also need this update? There are several other spots that use only `HTMLElement` that you correctly did NOT update because we know we are working with `HTMLElement` in those situations 👍